Cargo.toml

```diff
@@ -22,33 +22,29 @@ futures-sink = "0.3.30"
 futures-util = { version = "0.3.30", features = ["sink"] }
 hyper = { version = "0.14.16", features = ["full"], optional = true }
 pin-project = "1"
-quinn = { package = "iroh-quinn", version = "0.10", optional = true }
+quinn = { package = "iroh-quinn", version = "0.11", optional = true }
 serde = { version = "1.0.183", features = ["derive"] }
 tokio = { version = "1", default-features = false, features = ["macros", "sync"] }
 tokio-serde = { version = "0.8", features = ["bincode"], optional = true }
 tokio-util = { version = "0.7", features = ["codec"], optional = true }
 tracing = "0.1"
-quinn-udp = { package = "iroh-quinn-udp", version = "0.4.0", optional = true }
-interprocess = { version = "2.1", features = ["tokio"], optional = true }
 hex = "0.4.3"
 futures = { version = "0.3.30", optional = true }
 anyhow = "1.0.73"
 
-[dependencies.educe]
-# This is an unused dependency, it is needed to make the minimal
-# crates versions work since tokio-serde depends on this.  See
-# https://github.com/carllerche/tokio-serde/pull/29
-version = "0.4.20"
+# Indirect dependencies, is needed to make the minimal crates versions work
+educe = "0.4.20" # tokio-serde
+slab = "0.4.9" # iroh-quinn
```

Comment:
I don't think we need the extra `slab` in here anymore, do we?

Reply:
Or rather -  we won't, once we depend on the next iroh-quinn release.

Reply:
yeah, I don't want to cut a release for quinn, just for this